tests/stdlib/tlists.nim

```diff
@@ -81,3 +81,39 @@ block tlistsToString:
     l.append('2')
     l.append('3')
     doAssert $l == """['1', '2', '3']"""
+
+block SinglyLinkedListConversion:
+  let l: seq[int] = @[]
+  doAssert $l.toSinglyLinkedList == "[]"
```

Comment:
use toSeq for tests; in general it's more robust and independent of how something is rendered